pkg/apis/kubermatic/v1/audit_logging.go

```diff
@@ -53,4 +53,18 @@ type AuditLoggingSettings struct {
 	PolicyPreset AuditPolicyPreset `json:"policyPreset,omitempty"`
 	// Optional: Configures the fluent-bit sidecar deployed alongside kube-apiserver.
 	SidecarSettings *AuditSidecarSettings `json:"sidecar,omitempty"`
+	// Optional: Configures the webhook backend for audit logs.
+	WebhookBackend *AuditWebhookBackendSettings `json:"webhookBackend,omitempty"`
+}
+
+// AuditWebhookBackendSettings configures webhook backend for audit logging functionality.
+type AuditWebhookBackendSettings struct {
+	// Enabled will enable or disable audit webhook backend.
+	Enabled bool `json:"enabled,omitempty"`
+	// Optional: PolicyPreset can be set to utilize a pre-defined set of audit policy rules.
+	PolicyPreset AuditPolicyPreset `json:"policyPreset,omitempty"`
```

Comment:
PolicyPreset was included the AuditWebhookBackendSettings so that admins can specify the policy they want to configure for the logs to be sent to webhook backend. As per the current implementation the policy preset gets set to metadata if auditlogging (log backend) is not [enabled](https://github.com/kubermatic/kubermatic/blob/main/pkg/resources/apiserver/audit.go#L156) but if the admin wants to use only the webhook backend then also they need some way to specify the audit policy of their choice without enabling the audit logging.

I see the point that it shouldn't actually be part of the webhookbackendsettings, updated the code to use the policyPreset under auditlogging for both audit logging and audit webhook.